source/macros.tex

```diff
@@ -234,7 +234,7 @@
 \newcommand{\mname}[1]{\tcode{\unun\ungap#1\ungap\unun}}
 
 %% Ranges
-\newcommand{\Range}[4]{\tcode{#1\brk{}#3,\brk{}#4\brk{}#2}\xspace}
+\newcommand{\Range}[4]{\tcode{#1\brk{}#3, \brk{}#4\brk{}#2}\xspace}
```

Comment:
The change adds some line breaks inside ranges. Here's one example:

![image](https://cloud.githubusercontent.com/assets/6378233/11905543/fc6d4ff0-a5be-11e5-80b0-5e772cce3fb7.png)